resdk/resdk_logger.py

```diff
@@ -0,0 +1,223 @@
+"""
```

Comment:
Include this module docs in Reference (`docs/ref.rst`). Use `automodule`, see http://www.sphinx-doc.org/en/stable/ext/autodoc.html